ci-operator/step-registry/ipi/conf/vsphere/zones/multisubnets/ipi-conf-vsphere-zones-multisubnets-chain.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"path": "ipi/conf/vsphere/zones/multisubnets/ipi-conf-vsphere-zones-multisubnets-chain.yaml",
```

Comment:
Suggest to add one more folder `external-lb` to be `ipi/conf/vsphere/zones/multisubnets/external-lb`